addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/XxeScanRule.java

```diff
@@ -267,11 +269,58 @@ private void localFileReflectionAttack(HttpMessage msg) {
                             .setEvidence(matcher.group())
                             .setMessage(msg)
                             .raise();
+                    alertRaised = true;
                 }
                 if (isStop()) {
                     return;
                 }
             }
+
+            // Now if no issue is found yet, then we replace the values one at a time. Do this for a
+            // fixed
+            // number
+            // of Elements, depending on the strength at which the rule is used.
+            if (!alertRaised) {
+                int totalTagCount = getTotalTagCount(msg.getRequestBody().toString());
+                int maxValuesChanged = 0;
+
+                if (this.getAttackStrength() == AttackStrength.LOW) {
+                    maxValuesChanged = 36 / LOCAL_FILE_TARGETS.length;
+                } else if (this.getAttackStrength() == AttackStrength.MEDIUM) {
+                    maxValuesChanged = 72 / LOCAL_FILE_TARGETS.length;
+                } else if (this.getAttackStrength() == AttackStrength.HIGH) {
+                    maxValuesChanged = 144 / LOCAL_FILE_TARGETS.length;
+                } else if (this.getAttackStrength() == AttackStrength.INSANE) {
+                    maxValuesChanged = totalTagCount;
+                }
+
+                for (int tagIdx = 1;
+                        ((tagIdx <= totalTagCount) && (tagIdx <= maxValuesChanged));
+                        tagIdx++) {
+                    String requestBody =
+                            createTagSpecificLfrPayload(msg.getRequestBody().toString(), tagIdx);
```

Comment:
```suggestion
                    requestBody =
                            createTagSpecificLfrPayload(msg.getRequestBody().toString(), tagIdx);
```

This is causing the build to fail because `requestBody` was already declared earlier.